servicepulse/install-servicepulse-in-iis.md

```diff
@@ -37,6 +37,10 @@ It is possible to manually install ServicePulse using IIS following these steps:
 ServicePulse.Host.exe --extract --outPath="C:\inetpub\websites\ServicePulse"
 ```
 
+Note: `ServicePulse.Host.exe` can be found in the ServicePulse installation directory. The default location for this directory is `%programfiles(x86)%\Particular Software\ServicePulse`
+
+Once the ServicePulse files are successfully extracted, configure a new IIS website whose physical path points to the location where the files have been extracted. Configure it to use port `9090`.
```

Comment:
Bullet?

Reply:
@sergioc For cosmetic changes like this feel free to edit directly.

Reply:
What do you mean by "bullet"? @sergioc?

Reply:
@weralabaj it's not a cosmetic change. I want to verify with @WilliamBZA whether the content should be structured in a bullet point list.

> What do you mean by "bullet"?

@WilliamBZA Whether these paragraphs should be bullet points. There's a lone bullet point before the note and preceded by a paragraph suggesting a list of instructions:

> It is possible to manually install ServicePulse using IIS following these steps:
>
>* Extract the ServicePulse files using the following command at a command prompt:

Reply:
I'd actually rather remove the lone bullet then.  The bullets won't be copied over to the snippets and notes, so the alignment will look strange I think. Thoughts?

Reply:
Since it's a set of instruction steps, I think presenting as an actual list makes it easier to follow. Perhaps numbered steps?

Reply:
Numbered will also have the notes and snippets out of alignment :/

Reply:
The alignment probably requires docs engine adjustments. So the question now is - is visual alignment more important than content structure and readability?

Reply:
I would argue that having items out of line hinders readability, contrary to what you seem to be proposing.

I have removed the lone bullet point.